stack/stack-setup-2.yaml

```diff
@@ -512,6 +512,30 @@ ghc:
             content-length: 111330896
             sha1: 057a8af762d0259204cb0b6d758fc6b31f536d46
 
+    # OpenBSD releases are not compatible.
+    # So we need separate blocks for openbsd64-6.0, openbsd64-6.1, with any bindists that are there.
+    # Each block should be `openbsd64-$GhcBuild`, where $GhcBuild on OpenBSD is the major release.
+    # Worse, the `$GhcBuild` is not yet autodetected and should be specified by hand:
+    # https://docs.haskellstack.org/en/stable/yaml_configuration/#ghc-build
+    openbsd64-maj5-min9:
+        8.0.1:
+            url: "https://github.com/commercialhaskell/ghc/releases/download/ghc-8.0.1-release/ghc-8.0.1-x86_64-unknown-openbsd5.9.tar.xz"
+            content-length: 100619184
+            sha1: 7ed707de43456ddc4534f68a17ef58b7099e9a59
+        8.0.2:
+            url: "https://github.com/commercialhaskell/ghc/releases/download/ghc-8.0.2-release/ghc-8.0.2-x86_64-unknown-openbsd.tar.xz"
+            content-length: 100054832
+            sha1: 426b01e6ac72b5e039eb9626649506bc47eb9510
```

Comment:
This block could in theory be merged already (though it'd be unused and might bitrot) and should work once https://github.com/commercialhaskell/stack/pull/3184 is in.